arm-network/2016-12-01/swagger/virtualNetworkGateway.json

```diff
@@ -1600,6 +1611,130 @@
       ],	  
       "description": "Response for GetConnectionSharedKey API service call"
     },
+    "IpsecPolicy": {
+      "properties": {
+        "sALifeTimeSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The Security Association lifetime in seconds for a site to site VPN tunnel."
+        },
+        "sADataSizeKilobytes": {
```

Comment:
> If that is the case, then the entire network api is named incorrectly. The network service uses upper camel case. But I have done testing with the rest specs and the existing jsons do work with our service.

@henry416 - Please do not take a look at the C# code on the service side. That is what confuses new people all the time. Please open Fiddler and see the actual stuff on the wire and you will find that NRP is sending properties of the request body or the response body in camelCase.